packages/ts/examples/cluster-dashboard/src/styles.css

```
@@ -0,0 +1,246 @@
+@import url("https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500&family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap");
```

Stylelint (17.14.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500&family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500&family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap"" (import-notation)

(import-notation)